### ISSUE TITLE
Rename Extension: Brave to Extension: Brave Shields in task manager

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#6244]
## Description 

Rename Extension: Brave to Extension: Brave Shields in task manager

## Steps to Reproduce

   1. Open brave
   2. Open browser task manager

## Actual result:

![image](https://user-images.githubusercontent.com/17010094/65924841-84a0d380-e3bc-11e9-97c4-cd1cc628a962.png)

## Expected result:
Shields process should say `Extension: Brave Shields`

## Reproduces how often: 

Easy

## Brave version (brave://version info)

All

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the dev channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by rwhitman2017** [comment]
Does anyone have any ideas on where this might be so that it can be fixed? Me and my teammate have been searching around for awhile. Any help would be greatly appreciated.

**Comment by bsclifton** [comment]
@rwhitman2017 yes- it should be just as easy as changing https://github.com/brave/brave-core/blob/master/components/brave_extension/extension/brave_extension/manifest.json

There may be other places expecting the name to be `Brave` though; so we'd need to be confident that everything else works

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
